docs/index.html

```diff
@@ -53,6 +53,17 @@ <h2>Latest release: 6.2 (2021-09-29)</h2>
       <li><a class="Source" href="https://sourceforge.net/projects/mantid/files/6.2/mantid-6.2.0-Source.tar.xz/download">Source code</a></li>
     </ul>
     <a href="./archives.html" id="previous">Previous releases</a>
+  </section>
+  <section id="conda" class="wrapper">
+    <h2>Conda packages</h2>
+    <p>Mantid is available in a number of <a href="https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html">conda</a> packages via anaconda.org on the <a href="https://anaconda.org/mantid">mantid channel</a>.</p>
```

Comment:
The way it currently reads, the Conda link should point to our Mantid Conda channel or packages. You're discussing packages not Conda in general:

I would have something like:

---

Mantid is available from Conda [Conda Link] as a number of packages in the Mantid Channel [Channel link]

---

Also do we want to link to Conda or Mamba, since the former has licensing implications which we're not making explicit. Especially as people implicitly associate this project with FOSS

Reply:
So miniconda's main issues are around the default channels (aka on Anaconda.org, the main, anaconda and other default channels), which we are not utilizing here at all. So with regards to Licensing, we are good I think.

Users who are installing mantid, via conda, may already have conda installed (probably miniconda3) this is written in a way to support what we recommend and the other users who already have conda via another application.

I will offline discussion with you changes to the wording here.